### PR TITLE
Okta_v2 add pagination to okta-list-zones command

### PR DESCRIPTION
**Pull request by ostolero**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-1200)

## Description
Added support for pagination for okta-list-zones command.

### CODE REVIEW

File: Packs/Okta/Integrations/Okta_v2/Okta_v2.yml [hunk 1]
```diff
@@ -88,55 +113,113 @@ script:
...
+      isArray: false
+      secret: false
     description: Sets passwords without validating existing user credentials.
     execution: true
     name: okta-set-password
+    deprecated: false
   - arguments:
     - description: ID of the user to add to the group.
       name: userId
+      default: false
```
Comment (JudahSchwartz) [inline Packs/Okta/Integrations/Okta_v2/Okta_v2.yml:161]: these were all added from running format right?

File: Packs/Okta/Integrations/Okta_v2/Okta_v2.py [hunk 2]
```diff
@@ -587,12 +587,16 @@ def get_zone(self, zoneID):
...
         )
 
-    def list_zones(self):
+    def list_zones(self, limit):
         uri = 'zones'
-        return self._http_request(
-            method='GET',
-            url_suffix=uri
-        )
+        if limit:
```
Comment (JudahSchwartz) [inline Packs/Okta/Integrations/Okta_v2/Okta_v2.py:592]: remember to add to readme

**Review by JudahSchwartz — COMMENTED** [review]
looks great!

File: Packs/Okta/Integrations/Okta_v2/Okta_v2.yml (hunk 1 above)
Comment (ostolero) [reply Packs/Okta/Integrations/Okta_v2/Okta_v2.yml:161]: yes :)

**Review by JudahSchwartz — APPROVED** [review]

**Pull request merged**